activities/activity_laginhas.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+"""
+Comparing differences for resampling methods: average & median.
+"""
+import grass.script as gs
+
+
+def run_resamplediffs(scanned_elev, env, **kwargs):
+    raster_region = gs.region(env=env)
```

Comment:
Sorry, this is hopefully the last thing, this should have been the raster resolution:
```suggestion
    raster_region = gs.raster_info(scanned_elev)
```